data/default_hw_requirements.json

```diff
@@ -4,13 +4,17 @@
     "cpu_cores": 4,
     "ram_mib": 16384,
     "disk_size_gb": 120,
-    "installation_disk_speed_threshold_ms": 10
+    "installation_disk_speed_threshold_ms": 10,
+    "network_latency_threshold_ms": 100,
+    "packet_loss_percentage": 0
   },
   "worker": {
     "cpu_cores": 2,
     "ram_mib": 8192,
     "disk_size_gb": 120,
-    "installation_disk_speed_threshold_ms": 10
+    "installation_disk_speed_threshold_ms": 10,
+    "network_latency_threshold_ms": 500,
```

Comment:
it's a different value than what we have in the saas.yaml https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/assisted-installer/cicd/saas.yaml#L66
 Is it on purpose?

Reply:
Good catch. Should be the same. I'll change to 1000